pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories.md

```diff
@@ -0,0 +1,198 @@
+    </p>
+    <div class="content">
+        <amp-story-player style="width: 360px; height: 600px;">
+        <a
+        href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/"
+        style="--story-player-poster: url('https://amp.dev/static/samples/img/story_dog2_portrait.jpg')"
+        >
+        Stories in AMP - Hello World
+        </a>
+    </amp-story-player>
+    </div>
+    <p>
+        Such treat big ol pupper. Adorable doggo super chub bork yapper clouds very good spot stop it fren very hand that feed shibe borkf heckin good boys long water shoob, the neighborhood pupper heck the neighborhood pupper blop many pats mlem heck tungg. noodle horse. Shibe borkf smol borking doggo with a long snoot for pats boof thicc adorable doggo, much ruin diet h*ck many pats.
+    </p>
+    </body>
+</html>
+```
+[/example]
+
+## Page requirements
+
+Displaying an AMP story in a non-AMP page requires the use of the [`amp-story-player`](https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md) element. 
+
+### Import scripts
+
+Include the two required scripts in the head of your document:
+
+```html
+<script async src="https://cdn.ampproject.org/amp-story-player-v0.js">
+<link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel='stylesheet' type='text/css'>
+```
+
+The first script imports the logic for the player and the second one sets the default styling.  
+
+### Define HTML element
+
+Include the `<amp-story-player>` element inside the document `body`. 
+
+```html
+ <amp-story-player>
+ </amp-story-player>
+```
+
+### Specify the story
+
+Specify the desired story by placing an `<a>` tag inside of the `<amp-story-player>` element. Point the `href` to the story’s location. The `href` may point to the URL of a hosted story or a relative path. Place the title of the story within the `<a>` tags.
+
+```html
+ <amp-story-player style="width: 360px; height: 600px;">
+    <a
+      href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/">
+      Stories in AMP - Hello World
+    </a>
+  </amp-story-player>
+```
+
+### Size the player
+
+Avoid delays by defining the story player’s `width` and `height` as inline styles. But you may define the story player's width, height, and other styles as you would any other element's style.
+
+
+```html
+<body>
+...
+  <amp-story-player style="width: 360px; height: 600px;">
+...
+  </amp-story-player>
+...
+</body>
+```
+
+We recommend maintaining a 3:5 aspect ratio for the best user experience, but you may define any width and height. 
+
+#### Responsive sizing
+
+The story player's responsiveness works as any other block element. You may use media-queries or define percentages for width and height.
```

Comment:
I'd be good to offer an example for how to define a responsive block element maintaining the aspect ratio (it's not straightforward in CSS). Using percentages won't maintain the aspect ratio.

Reply:
cc @enriqe it's hard enough that maybe we should just make it a feature of the player (e.g. an attribute like `aspect-ratio="9:16"`)

Just a thought for the future.  Still probably makes sense to document the pure CSS way in the interim.

Reply:
@newmuis @Enriqe are either of you able to provide a code sample for this use case?

Reply:
Yep, I can put something together, i'll post it in a comment here in a bit

Reply:
Thanks so much @Enriqe - It's super appreciated!

Reply:
Sorry, forgot to come back to this 🙈...

Instead of setting a fixed width/height, publishers could specify something like this:
```
amp-story-player {
   /** To mantain a 3:5 ratio */
  width: 50vw;
  height: 83.35vw;
 }
```

Reply:
No worries - thanks!